docs/Plugin Architecture/Plugin Management.md

```diff
@@ -117,7 +117,7 @@ This command is less complicated than the install command. Basically all this co
 
 ##### Command Usage
 ```commandline
-cli plugin uninstall <plugin...>
+zowe plugins uninstall <plugin...>
```

Comment:
plug-in -- please hyphenate here and throughout the rest of the files in this PR

Reply:
In this case, we are giving an example of the command to run in the command line. For the CLI, the plugins command group is not hyphenated.

Reply:
@awharn - doh! my bad, you're right!